accessibility.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+order: 3
+title: "Accessibility"
+parent: "About the event"
+---
+
+We strive to reduce barriers that would prevent people from participating. If you have any access needs or questions, please contact [{{ site.email }}](mailto:{{ site.email }}) at least two weeks before the event.
+
+The details of this page are current as of {{ page.date }}.
```

Comment:
Is page.date showing up for you? It's blank locally. Maybe we need to build to see this?